setup.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+with open('requirements.txt') as f:
+    requirements = f.read().splitlines()
+    requirements = [req.replace('==', '>=') for req in requirements]
+
+# normal setuptool inputs
+setuptools.setup(
+    name='pollination-point-in-time-grid',                                     # will be used for package name
+    author='ladybug-tools',                                                 # the owner account for this package - required if pushed to Pollination
+    author_email='info@ladybug.tools',
+    packages=setuptools.find_namespace_packages(                            # required - that's how pollination find the package
+        include=['pollination.*'], exclude=['tests', '.github']
+    ),
+    install_requires=requirements,
+    use_scm_version=True,
+    setup_requires=['setuptools_scm'],
+    url='https://github.com/pollination/point-in-time-grid',                   # will be translated to home
+    project_urls={
+        'icon': 'https://raw.githubusercontent.com/ladybug-tools/artwork/master/icons_components/honeybee/png/dfrecipe.png',
```

Comment:
Hey @chriswmackey! I think you missed this one when you copied the files from daylight factor. :)